### DISCUSSION TITLE
Mountain Build Demonstration

### DISCUSSION THREAD

**Discussion by JWForever5504** [Mojang/minecraft-editor#60] [Show & Tell]
I managed to build a decent mountain in less than 10 minutes using editor mode!

Video quality is very bad. I took a 132 MB video, and made it smaller than 10 MB to upload it. Either way, here it is:
https://user-images.githubusercontent.com/127066569/224139773-6143c1bc-263b-4a1f-82d2-034b631c66cc.mp4

This shows just how easy it can be to make things that would have taken around an hour previously. And it's only going to get better!

**Comment by minecraftchloe** [comment]
So cool!!!!! Thanks for sharing!!
